demos/classification_demo/models.lst

```diff
@@ -50,6 +50,7 @@ resnet-50-pytorch
 resnet-50-tf
 resnet18-xnor-binary-onnx-0001
 resnet50-binary-0001
+rexnet-v1-1.0x
```

Comment:
I think it better to align names for width multiplier between different models (x1.0 in shufflenet used)

Reply:
Done